test/views/templates/ReturnObligationTemplateSpec.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.templates
+
+import java.time.LocalDate
+import models.Obligation
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import views.ViewBaseSpec
+
+class ReturnObligationTemplateSpec extends ViewBaseSpec {
+
+  "returnObligation template" when {
+
+    object Selectors {
+      val p1 = "p:nth-of-type(1)"
```

Comment:
Think about giving these selectors mor meaningful names e.g. `p2` could be renamed to `periodRangeDescription`